technical-reference/biztalk-settings-dashboard-group-page.md

```diff
@@ -51,7 +52,7 @@ Use the **Group** tab to modify the configuration of group-level performance set
 </tbody>
 </table>
 
-**Receive Location**
+**Receive Location** (New starting with BizTalk Server 2016)
```

Comment:
suggestion:
instead of putting it in brackets, you could 
- add a sentence in the next line, e.g. 'These settings are available starting with ...', or 
- a Note box.

Also, are the receive location settings available in 2016? I think even with the CU we didn't put the settings in the UI, right?